plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -66,6 +67,9 @@ var sampleConfig = `
   ## Set the interval to check if the Elasticsearch nodes are available
   ## Setting to "0s" will disable the health check (not recommended in production)
   health_check_interval = "10s"
+  ## Set the timeout for periodic health checks.
+  ## The default healthcheck timeout is 1s
+  health_check_timeout = "1s"
```

Comment:
In sample configuration we comment out the default as a project convention. This lets the user know what the setting will be even if they don't set it in the toml file. see https://github.com/influxdata/telegraf/blob/master/docs/developers/SAMPLE_CONFIG.md#style The commented default should be in the readme.md too.

The rest of the plugin doesn't seem to do this, but let's make the new setting follow the convention.

```suggestion
  #health_check_timeout = "1s"
```

Reply:
@aratik711 Please also reflect this in the README file.

And by doing so, the line saying `## The default healthcheck timeout is 1s` can now be left out, since that became obvious now.